config/foreman.hiera/mongo-redhat-7.yaml

```diff
@@ -0,0 +1,11 @@
+---
+mongodb::globals::server_package_name: 'rh-mongodb34-mongodb-server'
+mongodb::globals::client_package_name: 'rh-mongodb34-mongodb'
+mongodb::globals::service_name: 'rh-mongodb34-mongod'
+mongodb::server::config: '/etc/opt/rh/rh-mongodb34/mongod.conf'
+mongodb::server::pidfilepath: '/var/opt/rh/rh-mongodb34/run/mongodb/mongod.pid'
+mongodb::server::dbpath: '/var/lib/mongodb'
+mongodb::server::syslog: true
+mongodb::server::logpath: false
+mongodb::server::config_template: 'mongodb/mongodb.conf.2.6.erb'
+mongodb::server::storage_engine: 'wiredTiger'
```

Comment:
@chris1984 This will force a change on upgrades won't it? Thus breaking upgrades? Remind me how were mitigating that.